projects/storefrontlib/src/cms-components/asm/asm-session-timer/asm-session-timer.component.ts

```diff
@@ -0,0 +1,56 @@
+import { ChangeDetectorRef, Component, OnDestroy, OnInit } from '@angular/core';
+import { AsmConfig, RoutingService } from '@spartacus/core';
+import { Subscription } from 'rxjs';
+import { AsmComponentService } from '../asm-component.service';
+
+@Component({
+  selector: 'cx-asm-session-timer',
+  templateUrl: './asm-session-timer.component.html',
+})
+export class AsmSessionTimerComponent implements OnInit, OnDestroy {
+  private subscriptions = new Subscription();
+  private interval: any;
```

Comment:
any is ideal for this setInterval, however if you want to use the window.setInterview, we can set the type to number

Reply:
As discussed we'll leave it as is.